devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -96,6 +96,7 @@ metadata {
 
 		// Ajax Online
 		fingerprint manufacturer: "Ajaxonline", model: "AJ-RGBCCT 5 in 1", deviceJoinName: "Ajax Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K"
+        fingerprint profileId: "0104", manufacturer: "Ajax online Ltd", model: "AJ_ZB30_GU10", deviceJoinName: "Ajax Light"
```

Comment:
Please:
- correct spacing (4 spaces -> 1 tab)
- add a Raw Description of this device in a comment on the same line after the finger print
- since manufacturer and model are provided - profileId is not obligatory

What's the color temperature range of this device ?